src/XCCDF/item.h

```diff
@@ -344,6 +346,7 @@ struct xccdf_fixtext {
 
 struct xccdf_rule_result {
 	char *idref;
+	char *stig_rule_id;
```

Comment:
this seems excessive to store something this specific in every rule result. are you sure we can't do this in any different way? Can't we reference the actual Rule using idref and just get the ID there?

Reply:
That's a good ideia, it's done!